feature-libs/product/configurators/common/src/core/facade/configurator-group-status.service.ts

```diff
@@ -1,45 +1,51 @@
 import { Injectable } from '@angular/core';
 import { Store } from '@ngrx/store';
+import {
+  Configurator,
+  ConfiguratorActions,
+  ConfiguratorSelectors,
+  GenericConfigurator,
+  StateWithConfiguration,
+} from '@spartacus/core';
 import { Observable } from 'rxjs';
 import { take } from 'rxjs/operators';
-import { Configurator, GenericConfigurator } from '../../../model';
-import { StateWithConfiguration } from '../store';
-import * as ConfiguratorActions from '../store/actions/configurator.action';
-import { ConfiguratorSelectors } from '../store/selectors';
-import { ConfiguratorGroupUtilsService } from './configurator-group-utils.service';
+import { ConfiguratorFacadeUtilsService } from './utils/configurator-facade-utils.service';
 
 /**
- * Service for handling group status
+ * Service for handling group statuses
  */
 @Injectable()
 export class ConfiguratorGroupStatusService {
   constructor(
-    private store: Store<StateWithConfiguration>,
-    private configuratorGroupUtilsService: ConfiguratorGroupUtilsService
+    protected store: Store<StateWithConfiguration>,
+    protected configuratorFacadeUtilsService: ConfiguratorFacadeUtilsService
```

Comment:
What was the reason to rename from `ConfiguratorGroupUtilsService`?

`xxxFacadeUtilsService` looks a bit unnatural to me:
- we never used `facade` in the name of the service
- facade, utils and service are 3 type words in one class name

Reply:
Agree, I see. I wanted to express that it's not only about utilities for the group services, but will find a better name